context/app/static/js/components/Detail/Visualization/Visualization.jsx

```diff
@@ -1,22 +1,83 @@
-import React from 'react';
-import Paper from '@material-ui/core/Paper';
+import React, { useState, useEffect } from 'react';
 import { Vitessce } from 'vitessce';
+import Paper from '@material-ui/core/Paper';
+import Link from '@material-ui/core/Link';
+import ZoomOutMapIcon from '@material-ui/icons/ZoomOutMap';
 import SectionHeader from '../SectionHeader';
-import SectionContainer from '../SectionContainer';
-import 'vitessce/build-lib/es/production/static/css/index.css';
+import {
+  vitessceFixedHeight,
+  StyledSectionContainer,
+  ExpandButton,
+  TopSnackbar,
+  ExpandableDiv,
+  FlexContainer,
+  FlexBottomRight,
+  StyledFooterText,
+} from './style';
+import 'vitessce/dist/es/production/static/css/index.css';
 
 function Visualization(props) {
   const { vitData } = props;
 
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSnackbarOpen, setIsSnackbarOpen] = useState(false);
+
+  function handleExpand() {
+    setIsExpanded(true);
+    setIsSnackbarOpen(true);
+  }
+
+  function handleCollapse() {
+    setIsExpanded(false);
+    setIsSnackbarOpen(false);
+  }
+
+  useEffect(() => {
+    function onKeydown(event) {
+      if (event.key === 'Escape') {
+        handleCollapse();
+      }
+    }
+    window.addEventListener('keydown', onKeydown);
+    return () => {
+      window.removeEventListener('keydown', onKeydown);
+    };
+  }, []);
+
   return (
-    <SectionContainer id="visualization">
+    <StyledSectionContainer id="visualization">
       <SectionHeader variant="h3" component="h2">
         Visualization
+        <ExpandButton onClick={handleExpand}>
+          <ZoomOutMapIcon />
+        </ExpandButton>
       </SectionHeader>
       <Paper>
-        <Vitessce rowHeight={100} config={vitData} theme="light" />
+        <ExpandableDiv $isExpanded={isExpanded}>
+          <TopSnackbar
+            anchorOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+            open={isSnackbarOpen}
+            autoHideDuration={4000}
+            onClose={() => setIsSnackbarOpen(false)}
+            message="Press [esc] to exit full window."
+          />
+          <Vitessce config={vitData} theme="light" height={isExpanded ? null : vitessceFixedHeight} />
+        </ExpandableDiv>
       </Paper>
-    </SectionContainer>
+      <FlexContainer>
+        <FlexBottomRight>
+          <StyledFooterText>
+            Powered by&nbsp;
+            <Link href="http://vitessce.io" variant="body1" target="_blank">
```

Comment:
```suggestion
          <StyledFooterText variant="body2">
            Powered by&nbsp;
            <Link href="http://vitessce.io" target="_blank">
```

Reply:
Slightly larger (0.80rem vs 0.75rem), but good to take typography sizing from the theme.

Reply:
Also do we want to add `rel="noopener noreferrer"` to the Link?